highfidelity/hifi
# WL 21571 - Remove "common questions" button

> The button on the bottom of the tablet "common questions" doesn't launch or show anything when tapped by the stylus. Please remove it.

WL [21571](https://worklist.net/21571)

## Test Plan
- Open up the Tablet and Launch the Help App
- Confirm that the Bottom of the Window no longer has a "Common Questions" button on all the pages.
